README.md

```diff
@@ -0,0 +1,2 @@
+# Java Echo Server
+[![Build Status](https://travis-ci.org/junehoe/java-echo-server.svg?branch=master)](https://travis-ci.org/junehoe/java-echo-server)
```

Comment:
I know that the skeleton is pretty light right now, but could you update your README to include instructions for running the app and the tests? Someone unfamiliar with gradle might not know about the `gradle run` and `gradle test` commands. 

I liked seeing the build status badge on your README, but it's currently showing the build status as unknown. Any idea what might be causing that? I followed the link to Travis, and it's building successfully. 
![image](https://user-images.githubusercontent.com/6590135/68823167-d25d5c80-0647-11ea-8c2b-d608789cceeb.png)